wear/src/main/java/io/homeassistant/companion/android/complications/views/ComplicationConfigMainView.kt

```diff
@@ -172,9 +196,11 @@ fun PreviewMainConfigView() {
     MainConfigView(
         entity = simplifiedEntity,
         showTitle = true,
+        showUnit = true,
```

Comment:
Not 100% sure if it's the right thing to do, but I tried to make it enabled by default whenever a new complication is created, but it doesn't seem like this did the trick. Am I doing it wrong? Do we need to make it `true` all over the place in the code and add a post-migration query that sets it to `false` for existing complications?

Reply:
To enable it by default for new complications, simply change [the default of the state to `true`](https://github.com/home-assistant/android/pull/3703/files#diff-d8b3ef0c1255a9de6e1024cf3f0aee55bfc3b64cbf1775ba9e36701f38b5c2abR61).

Reply:
Ah, thanks. So is this one only used for the preview in Android Studio?

Reply:
Yes - that's why function name includes Preview and it has a `@Preview` annotation ;)

Reply:
Changed default state to `true` for new complications,